plugins/covid/lab.py

```diff
@@ -145,3 +213,48 @@ def positive(test):
     covid_test = _get_covid_test(test)
     result = get_result(test)
     return result in covid_test.POSITIVE_RESULTS
+
+
+def get_covid_result_ticker(patient):
+    """
+    Given a PATIENT, return a list of dictionaries representing the last
+    3 results for each test type, with a timestamp, result string, and test name.
```

Comment:
it looks like the last 2 results? Or am I reading it wrong

Reply:
it has to get to 3 to pass the > 2 test on line 233